meta-leda-components/recipes-sdv/packagegroups/packagegroup-sdv-additions.bb

```diff
@@ -20,7 +20,7 @@ SDV_EXTERNAL_DEPENDS = "\
     sdv-dapr-binaries \
     sdv-initialize-dapr-first-boot \
     sdv-helm-binaries \
-    sdv-container-databroker \
+    sdv-kuksa-val-databroker \
```

Comment:
Why is kuksa-val, instead of container

Reply:
The SDV-Container has been contributed to Eclipse Kuksa.VAL in the meantime, so we need to migrate to that.